docker-compose.yml

```diff
@@ -5,30 +5,41 @@ services:
     image: front-end-monorepo_dev:latest
     build:
       context: ./
-      target: bootstrap
     entrypoint:
       - "/bin/bash"
   fe-project:
-    image: front-end-monorepo_prod:latest
+    image: front-end-monorepo_dev:latest
     build:
       context: ./
-      target: production-apps
     entrypoint:
       - "yarn"
       - "workspace"
       - "@zooniverse/fe-project"
     command: ["start"]
+    environment:
+      - PORT=3000
+      - ASSET_PREFIX=http://localhost:3000
+      - NODE_ENV=production
+      - PANOPTES_ENV=production
```

Comment:
I think the Next runtime environment should match the environment that was used to build the classifier and apps, so I've changed these to both use `production`.

Reply:
The current values are here. https://github.com/zooniverse/front-end-monorepo/blob/50a9ceeccb037b14ce761ac9110d236e6e0302bc/packages/app-project/docker-compose-prod.yml#L12-L13

Reply:
the build env could be modified by injecting these ENV values as build args via compose definition. So we could have a development build for local dev image and the runtime envs would match this 

e.g. Caesar does this via dockerfile args and docker-compose build args https://github.com/zooniverse/caesar/blob/cb14a81b53f43b37025b0b109e5bd7c909402ea9/docker-compose.yml#L14-L17

Reply:
The classifier build environment is hard-coded in the webpack config.
https://github.com/zooniverse/front-end-monorepo/blob/71faab0da33610e5263d3b4e0b02b995980166d4/packages/lib-classifier/webpack.dist.js#L4-L8

So, we can easily inject a local build environment via compose, but the classifier build will ignore it and always use production. I think changing that would be a big enough change to our build process that it would merit its own PR and discussion.

If the classifier is always built in a production environment, I think that means the Next apps should be built and run in the same environment. Otherwise, I think we could get strange side-effects where SSR uses the staging API but client JS uses the production API.

I'm happy to add build args to this PR. I'm not 100% sure what the side effects of those would be, without spending some time to build and test different environments.

Reply:
I'm not sure. There's also a test, in the JS Panoptes client config, that fails when `PANOPTES_ENV=production`. There must be a reason the client is set up like that, but it's not clear if the test is wrong, or the client config is wrong. I found that when running the tests in a production image.